go.sum

```diff
@@ -1,9 +1,15 @@
+github.com/GianlucaGuarini/go-observable v0.0.0-20171228155646-e39e699e0a00 h1:4wp5bMTx9eV6in+ZKiUsyeOqYdp9ooqpw1YWXjwVHJo=
+github.com/GianlucaGuarini/go-observable v0.0.0-20171228155646-e39e699e0a00/go.mod h1:2pqNiwoZ8Fj1HBGWyPTXW/iPD332sJzTp3Iy0dIcFMc=
```

Comment:
Those 2 lines are wrong and should not have been merged

Reply:
If anybody does not care, I will remove those lines and merge to master directly.